src/System.Linq/src/System/Linq/Lookup.cs

```diff
@@ -182,7 +196,7 @@ internal List<TResult> ToList<TResult>(Func<TKey, IEnumerable<TElement>, TResult
             return _count;
         }
 
-        public IEnumerable<TResult> ApplyResultSelector<TResult>(Func<TKey, IEnumerable<TElement>, TResult> resultSelector)
+        public IEnumerator<TResult> ApplyResultSelector<TResult>(Func<TKey, IEnumerable<TElement>, TResult> resultSelector)
```

Comment:
This is a breaking change, our internal APICompat checks just tripped over this. I assume it was not intentional so I'm going to fix it internally.